maquetado/scss/titles.scss

```diff
@@ -0,0 +1,19 @@
+.main-title {
+  align-items: center;
+  display:  flex;
+  font-size: $text-big;
+  
+  &:before {
+    background: url('../assets/triangle.svg')  no-repeat;
+    content: '';
+    display:inline-block;
+    height:  $title-decoration-height;
+    left: 20px;
+    max-height: 100%;
+    max-width: 100%;
```

Comment:
why you need the `max-width` and `max-height`?

Reply:
I'm sorry, I confused a part of the theory of the style guide.